atomix/cluster/src/main/java/io/atomix/raft/partition/impl/RaftNamespaces.java

```diff
@@ -60,8 +58,8 @@
           .nextId(Namespaces.BEGIN_USER_CUSTOM_ID)
           .register(JoinRequest.class)
           .register(JoinResponse.class)
-          .register(LeaveRequest.class)
-          .register(LeaveResponse.class)
+          .register(Void.class) // LeaveRequest
```

Comment:
You can remove this entirely. We have already broken compatibility with the previous released version because of change in kryo version.